compiler/testData/codegen/boxInline/noInline/extensionReceiver.kt

```diff
@@ -1,3 +1,4 @@
+// IGNORE_BACKEND: JVM_IR
```

Comment:
If you understand why this is now failing, can you add a comment.

Reply:
It seems like its just a missing test in IrInlineCodegen. Fixed.